example/javascriptlib/testing/3-integration-suite-cypress/server/src/server.ts

```diff
@@ -0,0 +1,82 @@
+import * as http from 'http';
```

Comment:
Can this server code be shortened? Most of this looks like boilerplate 404/500/200 return code management that should be handled by some framework, and should not be something the user needs to write manually. Minimizing dependencies is good, but I think in this case it results in verbosity that gets in the way of understanding the example

Reply:
Sure

Reply:
I just replaced, it with express. Forgot that was an option.